gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalGcdHelper.java

```diff
@@ -107,7 +122,14 @@ public void start() throws IOException, InterruptedException {
     File gcdFolder = gcdPath.toFile();
     gcdFolder.deleteOnExit();
```

Comment:
Added some documentation.
Each instance of LocalGcdHelper is using a fresh copy to make sure no prior data is used (even
if prior tests that used it failed abnormally). The download happens only if the cached version (in temp folder) does not exists.  We could avoid the unzipping but this does not buy much for the tests and potentially increase pollution. Most tests that need it are using one instance per class.
see [here](https://github.com/GoogleCloudPlatform/gcloud-java/search?utf8=%E2%9C%93&q=LocalGcdHelper%5C.start%5C%28&type=Code).